content/feed.ttl

```diff
@@ -2,8 +2,10 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :feed
     skos:prefLabel "feed"@en;
+    skos:related "feedId";
```

Comment:
The related fields should not be strings. They are references `:feedId`

Reply:
> Can you give an example of what I did incorrectly vs what you think is correct? I dont understand this point.

This is an example.

Reply:
But not of how you think it ought to be represented.
I think previously you had something like `skos:related :"feedId";` but I wasn't convinced that this was the correct format.
documentation [here](https://www.w3.org/TR/skos-primer/#secpref:~:text=ex%3Aanimals%20rdf%3Atype%20skos%3AConcept%3B%0A%20%20skos%3AprefLabel%20%22animals%22%40en%3B%0A%20%20skos%3AaltLabel%20%22creatures%22%40en%3B%0A%20%20skos%3AprefLabel%20%22animaux%22%40fr%3B%0A%20%20skos%3AaltLabel%20%22cr%C3%A9atures%22%40fr.) makes it look like `skos:related "feedId";` is correct (granted it doesnt leverage the optional language tag).

Reply:
The anchor of the link is not working for me. 
Not sure where on the page you are .

From the source 
```
ex:birds rdf:type skos:Concept;
  skos:prefLabel "birds"@en;
  skos:related ex:ornithology.
ex:ornithology rdf:type skos:Concept;
  skos:prefLabel "ornithology"@en.
```
Line 1 is a triple: `subject relation object`. The object is a reference `skos:Concept` .

Our own domain is labeled `:` so the reference to orcfax's own feedId is `:feedId`